rotkehlchen/chain/evm/decoding/velodrome/balances.py

```diff
@@ -34,6 +52,63 @@ def __init__(
             deposit_event_types={(HistoryEventType.DEPOSIT, HistoryEventSubType.DEPOSIT_FOR_WRAPPED)},  # noqa: E501
             gauge_deposit_event_types={(HistoryEventType.DEPOSIT, HistoryEventSubType.DEPOSIT_ASSET)},  # noqa: E501
         )
+        self.protocol_token = protocol_token
+        self.voting_escrow_address = voting_escrow_address
 
     def get_gauge_address(self, event: 'EvmEvent') -> ChecksumEvmAddress | None:
         return event.address
+
+    def query_balances(self) -> BalancesSheetType:
+        balances = super().query_balances()
+        if len(addresses_with_deposits := self.addresses_with_activity(event_types=self.gauge_deposit_event_types)) == 0:  # noqa: E501
```

Comment:
```suggestion
        if len(addresses_with_deposits := self.addresses_with_deposits(products=None)) == 0:  # noqa: E501
```
Maybe should set `deposit_event_types` to `DEPOSIT_ASSET` and then just use the `addresses_with_deposits` function here. Or at least use the `addresses_with_gauge_deposits` function rather than `addresses_with_activity`